platform/broadcom/sonic-platform-modules-delta/debian/files

```diff
@@ -1 +1,2 @@
 platform-modules-ag9032v1_1.1_amd64.deb main extra
+platform-modules-ag9064_1.1_amd64.deb main extra
```

Comment:
this is debian generated file, please remove this file completed.

Reply:
I  will remove it